package.json

```diff
@@ -8,57 +8,76 @@
-    "inkscape": "^3.0.0",
+    "httperrors": "^2.3.0",
+    "impro": "~0.14.2",
+    "inkscape": "^3.1.1",
     "jpegtran": "^2.0.0",
     "mime": "^3.0.0",
     "optimist": "^0.6.1",
-    "optipng": "^4.0.0",
+    "optipng": "^4.2.0",
     "passerror": "^1.1.1",
     "pngcrush": "^3.0.0",
-    "pngquant": "^4.0.0",
-    "sharp": "^0.32.0"
+    "pngquant": "^4.2.0",
+    "sharp": "^0.33.2"
   },
   "devDependencies": {
-    "browser-sync": "^2.18.6",
-    "compression": "^1.6.1",
-    "eslint": "^8.1.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-config-standard": "^17.0.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-mocha": "^10.0.1",
-    "eslint-plugin-n": "^15.1.0",
-    "eslint-plugin-node": "^11.0.0",
-    "eslint-plugin-promise": "^6.0.0",
+    "browser-sync": "^3.0.2",
+    "compression": "^1.7.4",
+    "eslint": "^8.56.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-mocha": "^10.2.0",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-standard": "^5.0.0",
-    "express": "^4.16.3",
+    "express": "^4.18.2",
     "gifsicle": "^5.3.0",
-    "magicpen": "^6.0.2",
-    "magicpen-prism": "^5.0.0",
+    "magicpen": "^6.2.4",
+    "magicpen-prism": "^5.0.1",
     "memoizesync": "^1.1.1",
-    "mocha": "^9.0.1",
-    "nyc": "^15.0.0",
-    "offline-github-changelog": "^1.7.0",
-    "prettier": "~2.5.0",
-    "sinon": "^12.0.1",
-    "svgfilter": "^4.0.0",
-    "unexpected": "^12.0.0",
-    "unexpected-express": "^13.0.1",
-    "unexpected-http": "^8.0.1",
-    "unexpected-image": "^4.0.0",
-    "unexpected-sinon": "^11.0.0"
+    "mocha": "^10.2.0",
+    "nyc": "^15.1.0",
+    "offline-github-changelog": "^3.0.1",
+    "prettier": "^3.2.1",
+    "sinon": "^17.0.1",
+    "svgfilter": "^4.1.0",
+    "unexpected": "^12.0.5",
+    "unexpected-express": "^13.1.2",
+    "unexpected-http": "^9.0.0",
+    "unexpected-image": "^4.1.0",
+    "unexpected-sinon": "^11.1.0"
+  },
+  "overrides": {
+    "optimist": {
+      "minimist": "^0.2.3"
+    },
+    "lpad-align": {
+      "trim-newlines": "^3.0.1"
+    },
+    "semver-regex": "^3.1.4",
+    "gm-papandreou": {
+      "debug": "^2.6.9"
+    },
+    "cacheable-request": {
+      "http-cache-semantics": "^4.1.1"
+    }
   },
   "scripts": {
     "docker:build": "docker build -t express-processimage-dev .",
     "docker": "docker run --rm -it -v \"$(pwd):/home/nvm/express-processimage\" express-processimage-dev",
-    "lint": "eslint . && prettier --check '**/*.{js,md}'",
-    "test": "mocha",
+    "prettier:check": "prettier --check '**/*.{js,md}'",
+    "prettier:write": "prettier --write '**/*.{js,md}'",
+    "lint": "eslint . && npm run prettier:check",
+    "mocha": "mocha",
+    "mocha:inspect-brk": "node --inspect-brk ./node_modules/.bin/mocha",
+    "test": "npm run lint && npm run mocha",
```

Comment:
Please revert this bit, it's a standard snippet I have in all my packages.

I recommend setting up a prettier and an eslint plugin in your editor, that way you don't need to interact with `prettier` on the command line.

Reply:
Reverted the changes :+1: 

I find Prettier too opinionated and hence don't use it.

Reply:
Reverted the changes 👍

I find Prettier too opinionated and hence don't use it.

Reply:
You can still install the extension and configure it to only do something when the project is configured to use prettier via eg. a `.prettierrc` file at the root.

Reply:
> I find Prettier too opinionated and hence don't use it.

Trust me, it's super liberating to choose to not care about source formatting :innocent:

Reply:
I attempted to start using it two times, but didn't agree with its behavior for readability purposes. A bit like the opinion described by a developer in https://antfu.me/posts/why-not-prettier

I find ESLint formatting (now via [@stylistic/eslint-plugin](https://www.npmjs.com/package/@stylistic/eslint-plugin)) behavior good for my needs.

Reply:
To each their own :hugs: